src/Elements/Analysis/AnalysisMesh.cs

```diff
@@ -144,5 +131,35 @@ public void Tessellate(ref Mesh mesh)
                 }
             }
         }
+
+        /// <summary>
+        /// Compute a value for each grid cell.
+        /// </summary>
+        public void Analyze()
+        {
+            this._results = new List<Tuple<Polygon, double>>();
```

Comment:
May I suggest using C# 7's named tuple types so we don't have to contend with .Item1, .Item2 etc? So `new List<(Polygon cell, double analysisResult)>` or something?

Reply:
That’s a great idea.

Reply:
Done.